tokens-accounting/src/data.rs

```diff
@@ -99,25 +99,35 @@ pub struct FungibleTokenData {
     metadata_uri: Vec<u8>,
     total_supply: TokenTotalSupply,
     locked: bool,
-    reissuance_controller: Destination,
+    is_freezable: IsTokenFreezable,
+    is_unfreezable: IsTokenUnfreezable,
+    frozen: bool,
```

Comment:
Please add docs explaining what these really mean. These are really not that intuitive as you might think. Take another look at all these variables and think how people can misunderstand them. Every variable has a story behind it that should be told, so that code verification can be made easier.

Reply:
Please check in Boris's PR, there's a similar ambiguity that I mentioned.

Reply:
I reworked enums to be less ambiguous, take a look